perf-test/pod-startup/measure.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+NAMESPACE="serverless-workstream"
+DEPLOYMENT_NAME="vllm-test"
+LABEL="app=vllm-test"
+CLEANUP=${CLEANUP:-false}
+
+# For MacOS, `brew install coreutils` can make gdate available, if necessary
+START_TIME=$(gdate +%s%3N)
```

Comment:
```console
bash-3.2$ gdate
bash: gdate: command not found
```

Reply:
Hi @MikeSpreitzer , this script is supposed to be used from a Mac (which establishes VPN connection to intranet), from where are you trying this?